### PR TITLE
prevent deletion of sales invoices if country is Nepal

### PR DESCRIPTION
**Pull request by ghost**
The first of several compliance steps for Nepal. Prevents deletion of Sales Invoices in Nepal (using France's code as a model).

See this thread: https://discuss.erpnext.com/t/compliance-requirement-that-must-be-fulfilled-to-get-approval-from-tax-authority-in-nepal/34532/3

### CODE REVIEW

**Comment by theopen-institute** [comment]
Hmm...my apologies, but I don't understand why the unit test is failing. I've got a test method in utils.py  returning "overridden", but the test is still getting "original" with flag "Nepal". Any tips?

File: erpnext/hooks.py [hunk 1]
```diff
@@ -201,7 +201,7 @@
 	"Sales Invoice": {
 		'validate': 'erpnext.regional.india.utils.set_place_of_supply',
 		"on_submit": "erpnext.regional.france.utils.create_transaction_log",
-		"on_trash": "erpnext.regional.france.utils.check_deletion_permission"
+		"on_trash": ["erpnext.regional.france.utils.check_deletion_permission", "erpnext.regional.nepal.utils.check_deletion_permission"]
```
Comment (rmehta) [inline erpnext/hooks.py]: @theopen-institute thanks for the PR!

just make a common function and just add to the list of countries.

`erpnext.regional.check_delete_permission`

**Review by rmehta — CHANGES_REQUESTED** [review]

**Comment by nabinhait** [comment]
@theopen-institute I have fixed the PR and merged it. By the way, travis was failing because you did not mention `regional_overrides` in hooks.

**Comment by theopen-institute** [comment]
Thanks @nabinhait and @rmehta! I'm relatively new to frappe and python, but hopefully my next PR will have fewer problems :)

**Pull request closed without merging**